src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -3102,6 +3102,18 @@
     text: az bicep generate-params --file {bicep_file} --output-format {output_format} --include-params {include_params}
 """
 
+helps['bicep lint'] = """
+type: command
+short-summary: Lints a Bicep file.
+examples:
+  - name: Lints a Bicep file.
+    text: az bicep lint --file {bicep_file}
+  - name: Lints a Bicep file without restoring external modules.
+    text: az bicep lint --file {bicep_file} --no-restore
+  - name: Lints a Bicep file with specified diagnostics format. Valid values are ( default | sarif ).
+    text: az bicep lint --file {bicep_file} --diagnostics-format {diagnostics_format}
+"""
```

Comment:
It is better to keep consistence with other help messages and use first-person verbs.
```suggestion
helps['bicep lint'] = """
type: command
short-summary: Lint a Bicep file.
examples:
  - name: Lint a Bicep file.
    text: az bicep lint --file {bicep_file}
  - name: Lint a Bicep file without restoring external modules.
    text: az bicep lint --file {bicep_file} --no-restore
  - name: Lint a Bicep file with specified diagnostics format. Valid values are ( default | sarif ).
    text: az bicep lint --file {bicep_file} --diagnostics-format {diagnostics_format}
"""
```